docs/reference/src/advanced/style.md

```diff
@@ -1,54 +1,53 @@
+The styles available include:
 
-Before you start your compilation, you can select the style by setting the `SLINT_STYLE` variable
-to one of the style names, such as `fluent` for example.
+-   `fluent`, `fluent-light`, `fluent-dark`: These variants belong to the **Fluent** style, which is based on the [Fluent Design System](https://fluent2.microsoft.design/).
+-   `material`, `material-light`, `material-dark`: These variants are part of the **Material** style, which follows the [Material Design](https://m3.material.io).
+-   `cupertino`, `cupertino-light`, `cupertino-dark`: The **Cupertino** variants emulate the style used by macOS applications. (Experimental)
+-   `native`: The **Native** style mimics the controls native to the platform on which they are used. This includes specific support for the appearance of controls on macOS and Windows. This style requires Qt to be installed on your system.
 
-## Using the `slint_build` Crate
+Some styles have a `-light` and `-dark` variant, in addition to their standard variant.
+The standard variant will switch between the dark or light variant based on your desktop settings.
 
-Select the style with the [`slint_build::compile_with_config()`](https://docs.rs/slint-build/newest/slint_build/fn.compile_with_config.html) function in the compiler configuration argument.
+The widget style is determined at your project's compile time. The method to select a style depends on how you use Slint.
 
-## Using the `slint_interpreter` Crate
+If no style is selected, `native` becomes the default if Qt is installed. If `native` isn't available, `fluent` is selected as the default.
 
-Select the style with the [`slint_interpreter::ComponentCompiler::set_style()`](https://docs.rs/slint-interpreter/newest/slint_interpreter/struct.ComponentCompiler.html#method.set_style) function.
 
-</details>
+## Selecting a Widget Style with Rust:
 
-<details data-snippet-language="cpp">
-<summary>Selecting a Widget Style when using Slint with C++:</summary>
+You can choose the style before starting your compilation by setting the `SLINT_STYLE` variable to the name of your chosen style.
 
-Select the style by defining a `SLINT_STYLE` CMake cache variable to hold the style name as a string. This can be done for example on the command line:
+When using the `slint_build` API, utilize the [`slint_build::compile_with_config()`](https://docs.rs/slint-build/newest/slint_build/fn.compile_with_config.html) function.
 
-```sh
-cmake -DSLINT_STYLE="material" /path/to/source
-```
+When using the `slint_interpeter` API, employ the [`slint_interpreter::ComponentCompiler::set_style()`](https://docs.rs/slint-interpreter/newest/slint_interpreter/struct.ComponentCompiler.html#method.set_style) function.
+
+## Selecting a Widget Style when using C++
 
-</details>
+Define a `SLINT_STYLE` CMake cache variable to contain the style name as a string. This can be done, for instance, on the command line:
+
+cmake -DSLINT_STYLE="material" /path/to/source
```

Comment:
I don't know how to do this with the inline suggestion, but I think this should be surrounded by a code fence backtick series :)

Reply:
You can use more backticks to escape: 
````suggestion
```sh
cmake -DSLINT_STYLE="material" /path/to/source
```
````